replication-commands/src/main/java/org/codice/ditto/replication/commands/ConfigDeleteCommand.java

```diff
@@ -91,6 +115,41 @@
   private static final int DEFAULT_START_INDEX = 0;
 
   @Override
+  protected final Object executeWithSubject() throws Exception {
+    if (configNames == null || configNames.isEmpty()) {
+      if (deleteMetacards) {
+        removeAllReplicatedMetacards();
+        removeReplicationItems(null);
+      }
+
+      if (deleteHistory) {
+        removeHistoryItems(null);
+      }
+
+      if (deleteConfig) {
+        replicatorConfigLoader
+            .getAllConfigs()
+            .stream()
```

Comment:
✏️ No need for `.stream()` here since it was a collection being returned which supports `.forEach()` directly. Consider also adding `allConfigs()` which returns a stream to `ReplicatorConfigLoader` directly. This is preferable in APIs as it does not expose any collections or create temporary one and gives the freedom to clients to do what they want with the stream.

Reply:
Removed the stream() call but didn't update the ReplicatorConfigLoder